stdlib/out_channel.mli

```diff
@@ -0,0 +1,137 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           *)
+(*                                                                        *)
+(*   Copyright 2021 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Output channels.
+
+    @since 4.14.0 *)
+
+type t = out_channel
+(** The type of output channel. *)
+
+type open_flag = Stdlib.open_flag =
+  | Open_rdonly      (** open for reading. *)
+  | Open_wronly      (** open for writing. *)
+  | Open_append      (** open for appending: always write at end of file. *)
+  | Open_creat       (** create the file if it does not exist. *)
+  | Open_trunc       (** empty the file if it already exists. *)
+  | Open_excl        (** fail if Open_creat and the file already exists. *)
+  | Open_binary      (** open in binary mode (no conversion). *)
+  | Open_text        (** open in text mode (may perform conversions). *)
+  | Open_nonblock    (** open in non-blocking mode. *)
+(** Opening modes for {!open_gen}. *)
+
+val stdout : t
+(** The standard output for the process. *)
+
+val stderr : t
+(** The standard error output for the process. *)
+
+val open_text : string -> t
+(** Open the named file for writing, and return a new output channel on that
+    file, positioned at the beginning of the file. The file is truncated to zero
+    length if it already exists. It is created if it does not already exists. *)
+
+val open_bin : string -> t
+(** Same as {!open_text}, but the file is opened in binary mode, so that no
+    translation takes place during writes. On operating systems that do not
+    distinguish between text mode and binary mode, this function behaves like
+    {!open_text}. *)
```

Comment:
Same comment here, I don't expect channels do to funny business to my data. It's `open_text` that is weird not `open_bin`.

Reply:
Amended as suggested.